src/Filters/Filters.js

```diff
@@ -0,0 +1,175 @@
+import { useData } from "../Products";
+import { useState } from "react";
+import "./Filters.css";
+import { categories } from "../api/database";
+export const Filters = () => {
+  const {
+    sortBy,
+    showInventoryAll,
+    showFastDeliveryOnly,
+    priceRange,
+    dispatch,
+    level,
+    ratings,
+    searchString,
+    category,
+  } = useData();
+
+  console.log("18", sortBy);
+
+  const [search, setSearch] = useState("");
+
+  return (
+    <div className='filters'>
+      <h3>Search</h3>
+      <div className='input'>
+        <input
+          type='text'
+          onChange={(e) => {
+            return setSearch(e.target.value);
+            // return dispatch({ type: "SEARCH", payload: e.target.value });
+          }}
+          value={searchString ? searchString : search}
+          className='search-input-txt'
+          required
```

Comment:
this search bar is behaving little weird, like after searching any text when I try to edit the text its creating some difficulty

Reply:
Yup, thank you for pointing it out! I fixed it